src/apps/bridge/bridgePowerController.cpp

```diff
@@ -235,10 +235,11 @@ void BridgePowerController::_update(void) {
         } else { // Subsampling disabled
           stateLogPrintTarget("Sample", currentCycleS + sampleTimeRemainingS);
 #ifdef RAW_PRESSURE_ENABLE
-      if(!rbrPressureProcessorIsStarted()) {
-        rbrPressureProcessorStart();
-        printf("Started rbrPressureProcessor\n");
-      }
+          if (!rbrPressureProcessorIsStarted()) {
+            rbrPressureProcessorStart();
+            bridgeLogPrint(BRIDGE_SYS, BM_COMMON_LOG_LEVEL_INFO, USE_HEADER,
+                           "Started rbrPressureProcessor\n");
+          }
```

Comment:
When debugging this, it would have been nice to have "Started rbrPressureProcessor" in the SD card logs, so I changed this from `printf` to `bridgeLogPrint`.